pkg/actor/system.go

```diff
@@ -400,13 +459,21 @@ func (s *System) Start(ctx context.Context) {
 }
 
 // Stop the system, cancels all actors. It should be called after Start.
+// Messages sent before this call will be receive by actors.
```

Comment:
seems we need `StopNow`, sometimes we need it waste of time to send amessages if the whole system is stopped